packages/design-system/src/components/floating-panel/dialog.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { type ReactNode, type ComponentProps, type Ref } from "react";
```

Comment:
I think its confusing to put 2 completely unrelated components (dialog and popover) into floating-panel folder.

I think they can reuse the same styling while being separate components, not part of floating panel 

I assume Figma made you somehow do that, lets figure this out